### DISCUSSION TITLE
Where is the preferences menu item in 2.1?

### DISCUSSION THREAD

**Discussion by Dan-cer** [PintaProject/Pinta#759] [Q&A] [answered]
Thank you for your nice program! I'm using it since many years in Ubuntu and Linux Mint.
Now I mainly want to change the icon set of the latest version - 2.1. 
Unfortunately, I cannot find any setting options for this. Nowhere are there "Settings" in the menu? Have I overlooked something? I first installed Pinta 2.1 as an AppImage and then as a build from source. In both cases I can't find any settings.
Thank you in advance for your answer!

**Comment by cameronwhite** [comment, marked as answer]
Correct, there is not any setting for changing the icon theme so #555 would be the related issue

**Reply by Dan-cer** [reply]
Thank you for your quick answer. 
I'm fine with closing this discussion here to have it all in #555.
